### DISCUSSION TITLE
A big thanks

### DISCUSSION THREAD

**Discussion by anderskjeldsen** [playframework/play1#1471] [General]
I just wanted to say thank you to all the developers that keeps Play Framework 1 up to date. It entered maintenance mode more than 10 years ago, but I still think it kicks ass in 2024! So simple to use and so brilliantly designed. Not to mention the development pace, partly due to the fact that one doesn't have to rebuild the whole application to test the smallest change.

**Comment by blinder** [comment]
> I just wanted to say thank you to all the developers that keeps Play Framework 1 up to date. It entered maintenance mode more than 10 years ago, but I still think it kicks ass in 2024! So simple to use and so brilliantly designed. Not to mention the development pace, partly due to the fact that one doesn't have to rebuild the whole application to test the smallest change.

I completely agree 1000%. Seriously, a massive thank you to all of the contributors. I've been using Play since 1.1 days, and over the many years I've dabbled with "upgrading" to 2.x and have even poked 3 with a stick but still no one has managed to recreate the developer experience of Play1. 

So yes, much appreciation for our contributors here who keep the Play1 light shining bright!

**Comment by jvosloo** [comment]
Chiming in on this - it's 2024 and still nothing like Play1 out there 😱
We still love it and code on it daily. Sure we extended it a bit by layering on persistent jobs via Jobrunr for example, but our play1 apps just keep on trucking and the developer experience is still absolutely amazing, ever since we started using it in 2010. And since we added first intercooler.js and then went whole hog on htmx, we have SPA-like user experiences - everyone is happy 😃 

Great big thank you to the maintainers! 👏👊

**Comment by gaelforge** [comment]
Can I also say thanks to everyone who contributes here. No matter what other framework I try and use nothing beats the developer experience of Play 1. I'll even start new projects using it.

**Reply by blinder** [reply]
i just started a massive new project with it recently and i brought in a whole bunch new devs into the playframework orbit and they are like "wow, is it really this easy?"
